src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -291,13 +304,31 @@ func WithInitContainers(initContainers []corev1.Container) PodOption {
 	}
 }
 
+type PVCOption func(*corev1.PersistentVolumeClaim)
+
+// Option to add StorageClass
+func WithStorageClass(sc string) PVCOption {
+	return func(pvc *corev1.PersistentVolumeClaim) {
+		pvc.Spec.StorageClassName = &sc
+	}
+}
+
+// Option to add Persistent volume
+func WithPersistentVolume(pvName string) PVCOption {
+	return func(pvc *corev1.PersistentVolumeClaim) {
+		pvc.Spec.VolumeName = pvName
+	}
+}
 func NewPod(namespace string, podName string, containerName string, imageName string, options ...PodOption) *corev1.Pod {
 	runtimeClassName := "kata-remote"
 	annotationData := map[string]string{}
 
 	pod := &corev1.Pod{
 		ObjectMeta: metav1.ObjectMeta{Name: podName,
-			Namespace:   namespace,
+			Namespace: namespace,
+			Labels: map[string]string{
+				"app": podName,
+			},
```

Comment:
What's the reason for this? We have the `WithLabel` PodOption that can be passed in to achieve the same effect when it's required?